src/views/auth/Register.vue

```diff
@@ -116,12 +169,28 @@ export default {
         return
       }
 
-      if (this.submission.password !== this.submission.password_copy) {
+      if (!this.user.date_of_birth) {
+        this.errors.date_of_birth = ['Please enter your date of birth.']
+        return
+      }
+
+      if (this.user.password !== this.user.password_copy) {
         this.errors.password = ['Passwords do not match.']
         return
       }
 
-      this.axios.post(this.services['oms-core-elixir'] + '/signup/' + this.$route.params.id, this.submission).then(() => {
+      const body = _.pick(this.user, [
+        'username',
+        'password',
+        'first_name',
+        'last_name',
+        'email',
+        'date_of_birth',
+        'gender',
+        'phone'
+      ])
+
+      this.axios.post(this.services['oms-core-elixir'] + '/signup/' + this.$route.params.id, body).then(() => {
```

Comment:
Are we still having the backend at ‘oms-core-elixir’? Otherwise lgtm

Reply:
yep, for now. will be fixed at https://myaegee.atlassian.net/browse/MEMB-697